brands/tourism/hotel.json

```diff
@@ -231,6 +231,20 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|Grupo Barceló": {
+    "countryCodes": ["es"],
+    "tags": {
+      "brand": "Grupo Barceló",
+      "brand:en": "Barceló Group",
+      "brand:es": "Grupo Barceló",
+      "brand:wikidata": "Q15148996",
+      "brand:wikipedia": "es:Grupo Barceló",
```

Comment:
May you add as well the following tags, the original language of the firm?

```
"brand:ca": "Grup Barceló",
"brand:wikipedia": "ca:Grup Barceló",
"name:ca": "Grup Barceló",
```

Reply:
Added "ca" as original language

Reply:
Hi there. So usually with the index brands are only in two languages. English and then whatever the native language of the brand is. So is it ca or es? If its ca as @Crashillo say's it might be good to just go with that.

Reply:
Well, in such case, let's let it just in spanish. My bad. The actual name (brand name) is "Barceló Grupo", so I'd suggest:
```
      "brand": "Barceló grupo",
      "brand:en": "Barceló Group",
      "brand:es": "Grupo Barceló",
      "brand:wikidata": "Q15148996",
      "brand:wikipedia": "es:Grupo Barceló",
      "name": "Grupo Barceló",
      "name:en": "Barceló Group",
      "name:es": "Grupo Barceló",
      "tourism": "hotel"
```
Even though the native language was catalan, it's wide most-known as an spanish firm.

Reply:
"Barceló Grupo" would only be the website domain, If we take it in Spanish, then the correct name is "Grupo Barceló", in english "Barceló Group" and in catalan "Group Barceló"

So I can go with brand & brand:es as "Grupo Barceló" and brand:en with "Barceló Group", leaving brand:ca out